presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -711,6 +713,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         PARALLEL_PARSING_OF_PARTITION_VALUES_ENABLED,
                         "Enables parallel parsing of partition values from partition names using thread pool",
                         hiveClientConfig.isParallelParsingOfPartitionValuesEnabled(),
+                        false),
+                booleanProperty(
+                        QUICK_STATS_ENABLED,
+                        "Use quick stats to resolve stats",
```

Comment:
nit: I would make this more descriptive 

```suggestion
                        "Resolve some table statistics by reading parquet footers",
```

Reply:
This is not accurate. Parquet quick stats is one of the currently implemented ways, future implementations are an option for Hudi, Delta, Iceberg, ORC etc

Reply:
I see. Maybe a section in the documentation would be helpful?